File: app/src/main/java/com/spdroid/firestore/paging/demo/java/viewholder/PostViewHolder.java
package com.spdroid.firestore.paging.demo.java.viewholder;

import android.view.View;
import android.widget.TextView;
import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;
import com.spdroid.firestore.paging.demo.R;
import com.spdroid.firestore.paging.demo.java.model.Post;

public class PostViewHolder extends RecyclerView.ViewHolder {

    private TextView authorView;
    private TextView messageView;

    public PostViewHolder(@NonNull View itemView) {
        super(itemView);
        authorView = itemView.findViewById(R.id.post_AuthorName);
        messageView = itemView.findViewById(R.id.post_Message);
    }

    public void bind(Post post) {
        authorView.setText(post.authorName);
        messageView.setText(post.message);
    }

}

File: app/src/main/java/com/spdroid/firestore/paging/demo/kotlin/viewholder/PostViewHolder.kt
package com.spdroid.firestore.paging.demo.kotlin.viewholder

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.spdroid.firestore.paging.demo.R
import com.spdroid.firestore.paging.demo.kotlin.model.Post

class PostViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    private var authorView: TextView = itemView.findViewById(R.id.post_AuthorName)
    private var messageView: TextView = itemView.findViewById(R.id.post_Message)

    fun bind(post: Post) {
        authorView.text = post.authorName
        messageView.text = post.message
    }

}